### DISCUSSION TITLE
Feature Request - Select Field Option: Categories

### DISCUSSION THREAD

**Discussion by liamcrean** [ExpressionEngine/ExpressionEngine#3293] [Feature Requests]
Categories as an option for select field (in the same way it works for channels)

https://github.com/wesbaker/category_select

This addon works with EE6 (not tested EE7) but this really should be native to pro IMO. 

Very useful for outputting data on a per category basis like list of products and showing specific image based on category like colour, texture, material etc.

**Comment by intoeetive** [comment]
@liamcrean I'm not quite seeing the advantage of this VS the existing Categories field. It is just the UI - dropdown vs checkboxes that we have now?

**Comment by liamcrean** [comment]
@intoeetive 

This would be most useful when needing to categorise Grid content (images/media would be the most common use case, for products), for example. 

There's no category "field" as such in EE unless I'm massively mistaken? Just the option to categorise Entries?

**Reply by intoeetive** [reply]
Got it. Yes, completely makes sense
